physionet-django/console/templates/console/training_type/index.html

```diff
@@ -0,0 +1,71 @@
+{% extends "console/base_console.html" %}
+
+{% load static %}
+
+{% block title %}Training Types{% endblock %}
```

Comment:
Not sure Training Types is correct anymore

```suggestion
{% block title %}On Platform Courses{% endblock %}
```

or perhaps include the variable for the platform name here

Reply:
Added the platform name variable